### DISCUSSION TITLE
如何修改博客的主页（blogs index page）

### DISCUSSION THREAD

**Discussion by 66twz** [hexojs/hexo#5046] [Question / Answer] [answered]
我跟着知乎教程：https://zhuanlan.zhihu.com/p/366761432进行了设置，未能生效。
对
index_generator:
  path: /default-index/
  per_page: 10
  order_by: -date
中的任何参数进行修改，主页都是所有文章按照时间倒序排放。

谢谢解答！

**Comment by renbaoshuo** [comment, marked as answer]
修改 order_by 选项可以更改排序方式。
